README.md

```diff
@@ -74,7 +74,40 @@ The `useFloating` Svelte hook acts as a controller for all other Floating UI Sve
 
 ### useInteractions
 
-(tbd)
+The `useInteractions` Svelte hook allows you to consume multiple interactions. It ensures that event listeners from different hooks are properly registered instead of being overruled by one another.
+
+#### Usage
+
+```html
+<script>
+	import { useFloating, useHover, useFocus, useInteractions } from '@skeletonlabs/floating-ui-svelte';
```

Comment:
Nitpick: I'd order the imports by the general flow they're consumed:

1. useFloating
2. useInteractions
3. useX (events)